NAMESPACE

```diff
@@ -73,6 +73,7 @@ importFrom(GAMBLR.helpers,check_config_value)
 importFrom(GAMBLR.helpers,coding_vc)
 importFrom(GAMBLR.helpers,collate_qc_results)
 importFrom(GAMBLR.helpers,map_metadata_to_colours)
+importFrom(GAMBLR.helpers,rainfall_conv)
```

Comment:
Many of these functions are not (yet) available in GAMBLR.helpers. This might be something you are already aware of. Nonetheless, I wanted to bring it up in this review. These are the functions that get imported from GAMBLR.helpers by GAMBLR.viz, that are not (currently) available in GAMBLR.helpers:

```
coding_vc
collate_qc_results
map_metadata_colours
rainfall_conv
```

Reply:
Agree with Adam, only want to chime in that of these 4 suspects the 2 (coding_vc and rainfall_conv) made it to GAMBLR.helpers, while the other 2 did not because they will be part of other child(s)

Reply:
Sounds like it will be a crucial task to carefully look at the NAMESPACES for all the child repos (and main GAMBLR) and compare what gets imported and from where once all the child repos are configured adequately. As a step to ensure there are no conflicts in this.

Reply:
For sure, that is why I got every single object from GAMBLR (regardless of it export or not status) and we made a tracking document - so we can properly maintain what gets moved where and make sure nothing is lost in the process

Reply:
Exactly. For reference the tracking document can be found [here](https://docs.google.com/spreadsheets/d/1gaSqZBLRBDsloseiRvI9rUQ1K_rKD8fQ5wITJrKuDE8/edit#gid=0).